posts/2017-05-10-serverless-v1.13.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: New in Serverless v1.13 - Use values of async functions in your Serverless variables
+description: Async Serverless variables, Lambda tags, hello-world starter template and more in the Serverless Framework v1.13 release.
+date: 2017-05-10
+layout: Post
+authors:
+  - PhilippMuns
+---
+
+Today we're proud to announce the release of the Serverless Framework v1.13!
+
+Let's look into the new features and enhancements Serverless v1.13 introduces.
+
+## Highlights of 1.13.0
+
+**Note:** You can find a complete list of all the updates in the [changelog](https://github.com/serverless/serverless/blob/master/CHANGELOG.md).
+
+### Async value support for Serverless variables
+
```

Comment:
We should add list of config possibilities here:

- Connect variables to your teams secret management solution
- Fetch secrets from a remote database
- etc.